reconcile/test/cna/test_client.py

```diff
@@ -0,0 +1,45 @@
+from reconcile.cna.client import CNAClient
+
+
+def test_client_asset_type_metadata_init():
+    pass
```

Comment:
Placeholder?

Reply:
@fishi0x01 and i were discussing if we should have some hard validation of the metadata provided by the CNA module metadata endpoint with what we know about the modules in code. this is the placeholder test so we don't forget :)

Reply:
Worth a docstring or using [xfail](https://docs.pytest.org/en/7.1.x/how-to/skipping.html#xfail-mark-test-functions-as-expected-to-fail) to track?